src/shim/runtime/v2/rune/v2/create.go

```diff
@@ -0,0 +1,73 @@
+package v2
+
+import (
+	"context"
+
+	shimoci "github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/oci"
+	shimtypes "github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/types"
+	"github.com/containerd/containerd/runtime/v2/runc"
+	taskAPI "github.com/containerd/containerd/runtime/v2/task"
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/oci"
+	vc "github.com/kata-containers/kata-containers/src/runtime/virtcontainers"
+	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/compatoci"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*runc.Container, error) {
+	ociSpec, _, err := loadSpec(r.ID, r.Bundle)
+	if err != nil {
+		return nil, err
+	}
+	containerType, err := oci.ContainerType(*ociSpec)
+	if err != nil {
+		return nil, err
+	}
+
+	container, err := runc.NewContainer(ctx, s.platform, r)
+	if err != nil {
+		return nil, err
+	}
+
+	switch containerType {
+	case vc.PodSandbox:
```

Comment:
Hi, Mikko. Thank you for your advice. Could you please explain in detail how to use `cri-api`? 
This container type is referred to [kata runtime](https://github.com/confidential-containers/kata-containers-CCv0/blob/CCv0/src/runtime/pkg/containerd-shim-v2/create.go#L79).

Reply:
Looks like CRI's `type PodSandbox` is not what we want here, sorry. But I think this can be simplified so that there's no Kata dependency: looking at how Kata does it, it looks for a certain annotation in the OCI spec to check `config.json`is for a `PodSandbox` so that could be used directly here, right?

I noticed we have `loadSpec()` and `LoadSpec()` for doing the same thing.

Reply:
Thank you. Yes, In kata, it looks for a `"io.kubernetes.cri.container-type"` annotation in the OCI spec to check `config.json` is for a `PodSandbox`.   

Regarding whether to have Kata dependency, I have noticed [this issue](https://github.com/confidential-containers/enclave-cc/issues/15) about enclave-agent, and I agree that we should try to reuse code as much as possible in CoCo, but avoid copying it if at all possible.

![image](https://user-images.githubusercontent.com/26923638/180935424-a41a86fb-e43a-4f00-bf8f-0f98a7dafd3f.png)

Reply:
My thinking was that there's no need to reuse nor copy if we just check:

```suggestion
switch ociSpec.Annotations["io.kubernetes.cri.container-type"] {
	case "PodSandbox":
	...
	case "PodContainer":
	...
}
```

do you think that would work?